examples/example-project/.gitignore

```diff
@@ -0,0 +1,27 @@
+*.egg
+*.egg-info/
+*.pyc
+*.pyo
+*.sublime-*
+.coverage
+.env
+.gradle/
+.idea/
+.tox*
+.venv*
+/gradle
+/*/MANIFEST
+/*/activate
+/*/build/
+/*/config
+/*/coverage.xml
+/*/dist/
+/*/htmlcov/
+/*/mppy-pinned.txt
+/*/product-spec.json
+/build/
+/config/
+/dist/
+/ligradle/
+TEST-*.xml
+pinned.txt
```

Comment:
Remove because of the above change to `/*/*.pinned.txt` (or should it be just `/*pinned.txt)`?)